config.lua

```diff
@@ -1,4 +1,6 @@
 Config = {}
+Config.OxLoggingEnable = false -- See https://overextended.github.io/docs/ox_lib/Logger/Server
+Config.DiscordLoggingEnable = true
```

Comment:
IMO the naming is better when the verb is placed first, e.g `EnableDiscordLogging` sounds better.

Reply:
done